src/prefect/cli/config.py

```diff
@@ -207,6 +207,18 @@ def _process_setting(
         settings_output.append(f"{setting.name}='{display_value}'{source_blurb}")
         processed_settings.add(setting.name)
 
+    def _collect_defaults(default_values: Dict[str, Any], current_path: List[str] = []):
```

Comment:
just double checking, we're safe here with a mutable default because each time we call, we pass a new list right?